app/views/case_logs/_log_filters.erb

```diff
@@ -0,0 +1,32 @@
+<div class="app-filter-layout__filter" tabindex="-1" id="filter-panel">
+    <div class="app-filter" >
+        <div class="app-filter__header">
+            <h2 class="govuk-heading-m">Filters</h2>
+            <button 
+            class="app-filter__close" 
+            type="button" 
+            data-controller="filter"
+            data-action="click->filter#toggleFilter">
+            Close
+            </button>
+        </div>
+        <div class="app-filter__content">
+            <div class="govuk-form-group app-filter__group">
+                <%= form_with url: "/logs/filter", html: { method: :post }, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
```

Comment:
Don't think you need to specify the form builder here since the govuk one is set as the default anyway.
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/config/initializers/default_form_builder.rb